GpsDataAnalyzer/gpsdataset/gpsdataset.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+
+@dataclass
+class GpsData:
+	"""A GPS Data class"""
+	latitude: str
+	longitude: str
+	altitude: str
+	speed: float
+	time: str
```

Comment:
latitude, longitude, altitiude: should should probably be floats as well
time: probably want this to be a python datetime.datetime object not a string, that way we can easily do operations on the timestamps to see differences between two times or add time

Reply:
Do you feel is there any chance we will use the timezone UTC in the comparison?